### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by whay-wang-teo-credera-co-uk**

### CODE REVIEW

File: .githooks/post-checkout [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+git submodule update --remote .githooks
```
Comment (ChrisScottThomas) [inline .githooks/post-checkout:3]: githooks shouldn't be created like this, they are a submodule

File: .githooks/pre-commit [hunk 2]
```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```
Comment (ChrisScottThomas) [inline .githooks/pre-commit:1]: githooks shouldn't be created like this, they are a submodule

File: .gitmodules [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[submodule ".githooks"]
```
Comment (ChrisScottThomas) [inline .gitmodules:1]: githooks shouldn't be created like this, they are a submodule

File: .githooks/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# dataworks-githooks
```
Comment (ChrisScottThomas) [inline .githooks/README.md:1]: githooks shouldn't be created like this, they are a submodule.  They should be created by the `initial-commit` script when creating repos, or manually using the method in the `initial-commit` script.  Unsure why you don't have them already, perhaps the sample you used doesn't include the initial commit script? 🤔

Comment (whay-wang-teo-credera-co-uk) [reply .githooks/README.md:1]: no... I'm following the example in Dataworks-repo-template-docker
is that the wrong place for reference or?

**Review by ChrisScottThomas — APPROVED** [review]

**Pull request merged**